server/modules/allocations.js

```diff
@@ -1,5 +1,8 @@
 import axios from 'axios';
 import redis from 'redis';
+import dotenv from 'dotenv';
```

Comment:
You don't have to import this module, it has already been imported in `index.js`, given this file will still be linked to `index.js`, this import will seem redundant.

Reply:
Thanks for that. Since the module has not yet been imported into `index.js` at the moment, I feel it makes sense to import the `dotenv` here for now because it is needed for the functioning of the feature implemented.